### PR TITLE
[ MB-16105 ] Display additional MTO Compliance info to office UI

### PR DESCRIPTION
**Pull request by mmayo-truss**
## Summary

This PR displays the information required for MTO compliance in the Office UI. 

See [this figma link](https://www.figma.com/file/5AhuEkaH3bJVrtffU3wp22/Milmove-Office-App?type=design&node-id=11507-120468&mode=design&t=70uvHDDYnsj5Ekyj-0) for the design requirements.

Locally we are not using the real Contract number, so you will see a placeholder number if you run it on your machine.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the Office UI as a TOO or TIO
2. Go to a Move and click on the "Move task order" tab.
3. You should see the new information

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/about/supported-browsers) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/about/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

## Screenshots
New info in the header (now with 100% more right aligned Financial review links)
image

New info in the footer: 
image

### CODE REVIEW

**Comment by ronaktruss** [comment]
Based on the screenshot of the header we may need to add some additional stylings to ensure that the `Flag for financial review` link is right aligned.

**Comment by mmayo-truss** [comment]
> Based on the screenshot of the header we may need to add some additional stylings to ensure that the `Flag move for financial review` link is right aligned.

Good 👀  Nice catch

**Comment by mmayo-truss** [comment]
@ronaktruss I modified the financial review link to be right aligned. Thanks for catching that!

**Review by dinabeigelman — APPROVED** [review]
LGTM! Thanks for tweaking the page title alignment and adding space below the footer!

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx [hunk 1]
```diff
@@ -1220,6 +1221,12 @@ export const MoveTaskOrder = (props) => {
               </ShipmentContainer>
             );
           })}
+          <div className={styles.pageFooter}>
+            <div className={styles.pageFooterDetails}>
+              <h6 data-testid="packingAndShippingInstructions">{order?.packingAndShippingInstructions}</h6>
+              <h6 data-testid="methodOfPayment">{order?.methodOfPayment}</h6>
```
Comment (ronaktruss) [inline src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx:1227]: Are there any other alternatives to using `data-testid`? I think we had been moving away from using that unless needed in favor of the RTL philosophy where we try and test things the way that a user would interact with the page.

File: src/pages/Office/TXOMoveInfo/TXOTab.module.scss [hunk 2]
```diff
@@ -95,6 +95,28 @@
...
 
+.pageFooter {
+  display: flex;
+  align-items: center;
+  @include u-margin-top(4);
+
+  h6 {
+    margin: 0;
+    line-height: 1.25;
+    text-transform: none !important;
```
Comment (ronaktruss) [inline src/pages/Office/TXOMoveInfo/TXOTab.module.scss:106]: Do we need the `!important` here?

File: src/pages/Office/TXOMoveInfo/TXOTab.module.scss [hunk 3]
```diff
@@ -95,6 +95,28 @@
   }
 }
 
+.pageFooter {
+  display: flex;
+  align-items: center;
```
Comment (ronaktruss) [inline src/pages/Office/TXOMoveInfo/TXOTab.module.scss:100]: What does this do here?

**Review by ronaktruss — APPROVED** [review]
Works and functions as expected 🎉 ! Left a couple comments about potential enhancements/questions.

File: pkg/services/move/move_fetcher_test.go [hunk 4]
```diff
@@ -28,6 +28,7 @@ func (suite *MoveServiceSuite) TestMoveFetcher() {
 		suite.Equal(expectedMove.Status, actualMove.Status)
 		suite.Equal(expectedMove.AvailableToPrimeAt, actualMove.AvailableToPrimeAt)
 		suite.Equal(expectedMove.ContractorID, actualMove.ContractorID)
+		suite.Equal(expectedMove.Contractor.Name, actualMove.Contractor.Name)
```
[the remark this answers is not in the document]
Comment (mmayo-truss) [reply pkg/services/move/move_fetcher_test.go:31]: No particular reason. I just needed to check that the eager preload was actually preloading the contractor.

File: src/pages/Office/TXOMoveInfo/TXOTab.module.scss (hunk 2 above)
Comment (mmayo-truss) [reply src/pages/Office/TXOMoveInfo/TXOTab.module.scss:106]: I just checked and I don't think we do. Good callout!

File: src/pages/Office/TXOMoveInfo/TXOTab.module.scss (hunk 3 above)
Comment (mmayo-truss) [reply src/pages/Office/TXOMoveInfo/TXOTab.module.scss:100]: I believe that you are correct. This is unnecessary.

File: src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx (hunk 1 above)
Comment (mmayo-truss) [reply src/pages/Office/MoveTaskOrder/MoveTaskOrder.jsx]: Good callout. I removed the `data-testid` references and changed the test to simply check the page for the text.

**Pull request merged**